frontend/src/theme/components/Tag.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export type TagVariants = 'solid' | 'light'
+
+export const Tag: ComponentMultiStyleConfig = {
+  parts: [],
+  variants: {
```

Comment:
looking at the figma, there is a 4px padding on all sides.
the textStyle should also be `caption-1`

Reply:
changed! there's a point to note wrt the 4px padding - chakra's internal padding means that this isn't consistent throughout.

the 2 ways which we can go about doing this is to either add a custom key onto the `size` property within the styling or to override chakra's `size` theming uniformly from `sm` to `lg`. 

because our figma only shows a single size, i've opted for the former (preserves chakra functionality wrt sizing) but depending on what pearly says wrt sizing, i might opt for the second one (will confirm with her if the padding should change/be constant if this is chosen).

Reply:
just add a custom key, we only style for `md` right now

Reply:
yeppers, did that!